name.abuchen.portfolio/src/name/abuchen/portfolio/online/QuoteFeed.java

```diff
@@ -51,4 +51,6 @@
     List<LatestSecurityPrice> getHistoricalQuotes(String response, List<Exception> errors);
 
     List<Exchange> getExchanges(Security subject, List<Exception> errors);
+
+    List<Exchange> getRegions(Security subject, List<Exception> errors);
```

Comment:
Da wir ja jetzt Java8 nutzen, könnte man hier einen Default Implementierung mit "Collections.emptyList()" hinterlegen. Dann muss man die anderen QuoteFeeds nicht anpassen.

Reply:
Ich mache mich mal auf die Suche bei Google, ob ich hierzu für mich etwas finde.

Reply:
Ungefähr so:

```
default List<Exchange> getRegions(Security subject, List<Exception> errors) {
    return Collections.emptyList();
}
```

Dann brauchen die anderen Klassen die Methode nicht mehr implementieren.

Reply:
Statt eine neue Methode #getRegions einzuführen, habe ich jetzt einfach #getExchanges für den Eurostat Feed implementiert. Das macht die weitere Logik viel einfacher - z.b. in der AbstractQuoteFeedProviderPage muss man nur ändern, dass ein weiterer Feed auch Exchange unterstützt.

Vielleicht sollten wir #getExchanges bei Gelegenheit umbenennen - es ist ja eher so was wie "gib mir Feldhilfe für das tickerSymbol". Das können dann Börsenplätze (aber auch ganz andere Ticker sein) oder eben die Regionen.